signup-front/src/components/molecules/ContactTable.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+const contacts = [
+  { name: 'AidantConnect', address: 'contact@aidantsconnect.beta.gouv.fr' },
+  { name: 'Hubee', address: '' },
+  {
+    name: 'FranceConnect',
+    address: 'support.partenaires@franceconnect.gouv.fr',
```

Comment:
the address attribute is already set here : https://github.com/betagouv/datapass/blob/20b5f08bf4a946b9df78af0105ec1e3dfd1bd7db/signup-front/src/pages/FranceConnect.js#L133-L133 and in several other api pages. This information must not be duplicated.